### PR TITLE
Adds /adropammo

### PR DESCRIPTION
**Pull request by Dimiikou**

### CODE REVIEW

File: src/main/java/de/fuzzlemann/ucutils/commands/ADropAmmo.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+public class ADropAmmo implements TabCompletion{
+
+    private final Timer timer = new Timer();
+    private int ammo;
+
+    @Command(value = "adropammo", usage = "/%label% [Waffe] [Munition]")
+    public boolean onCommand(UPlayer p, String weapon, int ammo) {
+        if (!weapon.equalsIgnoreCase("M4") && !weapon.equalsIgnoreCase("MP5") && !weapon.equalsIgnoreCase("Jagdlinte") && !weapon.equalsIgnoreCase("Pistole")) return false;
+
+        if (!(ammo > 0)) return false;
```
Comment (paulzhng) [inline src/main/java/de/fuzzlemann/ucutils/commands/ADropAmmo.java:25]: could be simplified to ammo <= 0

File: src/main/java/de/fuzzlemann/ucutils/commands/ADropAmmo.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+ * @author Dimiikou
+ */
+public class ADropAmmo implements TabCompletion{
+
+    private final Timer timer = new Timer();
+    private int ammo;
+
+    @Command(value = "adropammo", usage = "/%label% [Waffe] [Munition]")
+    public boolean onCommand(UPlayer p, String weapon, int ammo) {
+        if (!weapon.equalsIgnoreCase("M4") && !weapon.equalsIgnoreCase("MP5") && !weapon.equalsIgnoreCase("Jagdlinte") && !weapon.equalsIgnoreCase("Pistole")) return false;
```
Comment (paulzhng) [inline src/main/java/de/fuzzlemann/ucutils/commands/ADropAmmo.java:23]: pronunciation mistake: "Jagdflinte"

File: src/main/java/de/fuzzlemann/ucutils/commands/ADropAmmo.java [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+        if (!(ammo > 0)) return false;
+
+        ADropAmmo instance = this;
+        instance.ammo = ammo;
+
+        timer.scheduleAtFixedRate(new TimerTask() {
```
Comment (paulzhng) [inline src/main/java/de/fuzzlemann/ucutils/commands/ADropAmmo.java:30]: add AtomicBoolean ensuring there cannot be two /adropammo commands at the same time

File: src/main/java/de/fuzzlemann/ucutils/commands/ADropAmmo.java [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+import java.util.TimerTask;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * @author Dimiikou
+ */
+public class ADropAmmo implements TabCompletion{
+
+    private final Timer timer = new Timer();
+    private int ammo;
```
Comment (paulzhng) [inline src/main/java/de/fuzzlemann/ucutils/commands/ADropAmmo.java:19]: doesn't need to be stored in the class, can be stored in the TimerTask directly

File: src/main/java/de/fuzzlemann/ucutils/commands/ADropAmmo.java [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+                    cancel();
+                }
+            }
+        }, 0, TimeUnit.SECONDS.toMillis(1));
+
+        return true;
+    }
+
+    public List<String> getTabCompletions(UPlayer p, String[] args) {
+        return Arrays.asList("M4", "MP5", "Jagdflinte", "Pistole");
```
Comment (paulzhng) [inline src/main/java/de/fuzzlemann/ucutils/commands/ADropAmmo.java:46]: Please put this List as a class variable and use the same List when checking for the command argument. Remember to do this case-insensitive

**Review by paulzhng — CHANGES_REQUESTED** [review]

**Comment by Dimiikou** [comment]
I'm going to create a new pr because i messed up sth

**Pull request closed without merging**